src/templates/fr/2020/methodology.html

```diff
@@ -2,20 +2,20 @@
 
 {% block title %}Méthodologie | Web Almanac par HTTP Archive{% endblock %}
 
-{% block description %}Décrit comment le Web Almanac 2020 a été conçu : Les ensembles de données et outils utilisés et la façon dont le projet a été exécuté.{% endblock %}
+{% block description %}Décrit comment le Web Almanac 2020 a été conçu&nbsp;: Les ensembles de données et outils utilisés et la façon dont le projet a été exécuté.{% endblock %}
```

Comment:
```suggestion
{% block description %}Décrit comment le Web Almanac 2020 a été conçu : Les ensembles de données et outils utilisés et la façon dont le projet a été exécuté.{% endblock %}
```

No need for a `&nbsp;` in the meta description.